src/Microsoft.ML.Dnn/DnnCatalog.cs

```diff
@@ -83,6 +83,120 @@ public static class DnnCatalog
             return new DnnRetrainEstimator(env, options, DnnUtils.LoadDnnModel(env, modelPath, true));
         }
 
+        /// <summary>
+        /// The options for the <see cref="ImageClassificationTransformer"/>.
```

Comment:
This should say it is the options for the Estimator, not the Transformer. These options are used to create the Estimator.